src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,112 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.RegistrationException;
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void register_userNotNull_ok() {
+        User userTest = new User();
+        userTest.setId(123456L);
+        userTest.setLogin("bondorol");
+        userTest.setPassword("123456");
+        userTest.setAge(18);
+        assertNotNull(userTest);
```

Comment:
What are you tesing here? I can not see any RegistrationService call in this test.